.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 # Reason for This PR
 
-`[Author TODO: add issue # or explain reasoning.]`
+`[Author TODO: add issue #.]`
+`[If there is no issue which states the need for this PR, create one first.]`
```

Comment:
I find that this statement is a bit disconnected with what's required in the checkbox below: `The issue which led to this PR has a clear conclusion`.

I'd rephrase this as something along the lines of `If there is no issue which states the need for this PR, create one and continue the PR after the issue has a clear conclusion.`.

Reply:
I think a big benefit would be to have contributors NOT open the PR until they both create an issue and reach a conclusion there. Creating a PR and an issue at the same time could likely lead to wasted work from the contributor, as there may be an entirely different approach agreed on the issue, and the initial PR variant has to be scrapped.

That being said, we could rephrase to `Open the PR after the related issue has a clear conclusion. If there is no issue which states the need for this PR, create one first.`

This phrasing also covers the case where an issue was opened, but it has no conclusion yet.

Reply:
> That being said, we could rephrase to Open the PR after the related issue has a clear conclusion. If there is no issue which states the need for this PR, create one first.

Sounds good :)